src/Symfony/Component/Notifier/Bridge/MessageBird/MessageBirdTransport.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MessageBird;
+
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Vasilij Duško <vasilij@prado.lt>
+ */
+final class MessageBirdTransport extends AbstractTransport
+{
+    protected const HOST = 'rest.messagebird.com';
+
+    private $authToken;
+    private $from;
+
+    public function __construct(string $authToken, string $from, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null)
+    {
+        $this->authToken = $authToken;
+        $this->from = $from;
```

Comment:
Hello, just a question why not let the originator as in the API to have the same naming in the API call ?

Reply:
@zairigimad Do you want to use parameter names the same as in the .env.local file?

Reply:
I mean 
```suggestion
        $this->originator = $originator;
```

Reply:
Good question. Few times changed from $this->phone to $this->sender and than $this->sender to $this->from https://github.com/symfony/symfony/pull/40607 :)))

Reply:
ah ok, well sender and from are good too :)

Reply:
From is more common in other notifier bridges

Reply:
We will have in few days new one bridge :))) only need to understand why in new branche I have lightsms code :)